hr_timesheet_purchase_order/tests/test_create_timesheet_po_recurrence.py

```diff
@@ -0,0 +1,622 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import date, datetime
+
+from dateutil.rrule import FR, MO, SA, TH
+from freezegun import freeze_time
+
+from odoo import _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import Form
+
+from .common_po_recurrence import TestTimesheetPOrecurrenceCommon
+
+
+class TestTimesheetPOrecurrence(TestTimesheetPOrecurrenceCommon):
```

Comment:
Would you add docstrings and assertions messages

Reply:
okay, I'll add it